uui-components/src/overlays/Dropdown.tsx

```diff
@@ -63,6 +66,7 @@ export class Dropdown extends React.Component<DropdownProps, DropdownState> {
         opened: this.props.value || false,
         bodyBoundingRect: { y: null, x: null, height: null, width: null },
         closeDropdownTimerId: null,
+        openDropdownTimerId: null,
```

Comment:
I think it's a good idea not to set openDropdownTimerId to components state and move it to instance variable instaed. So that: 
- we avoid unnecessary re-renders when timer id is changed
- we avoid potential issues with because setState is asynchronous and the changes are not applied immediately.

```
private openDropdownTimerId: NodeJS.Timeout = null;

```